README.md

```diff
@@ -201,23 +201,15 @@ You can configure Neovim to automatically run `:PackerCompile` whenever `plugins
 [an autocommand](https://neovim.io/doc/user/autocmd.html#:autocmd):
 
 ```
-augroup packer_user_config
-  autocmd!
-  autocmd BufWritePost plugins.lua source <afile> | PackerCompile
```

Comment:
is `source <afile>` not needed anymore?

Reply:
I'm not familiar with what `source <afile>` does. If I remember correctly I tried to find the equivalent in the lua api and there was nothing equivalent (but maybe I remember that wrong). I have the lua code in my init.lua as I wrote in this PR and it works.

Reply:
There is -- Lua callbacks get passed an `args` table, with `args.file` being equivalent to `<afile>`.

See `:h nvim_create_autocmd()`.